lib/dug/runner.rb

```diff
@@ -33,6 +33,7 @@ def unprocessed_notifications(use_cache: true)
         @unprocessed_notifications = nil
       end
       unprocessed_label = servicer.labels(use_cache: use_cache)[Dug.configuration.unprocessed_label_name]
+      raise "Create the label '#{Dug.configuration.unprocessed_label_name}' on gmail" unless unprocessed_label
```

Comment:
Yes. I hadn't yet gotten to more user friendly 'auto create' stuff and didn't know how I felt about it. 

One thought for people who *cough* might want to implement this for us *cough*: Dug should _not_ auto create labels for you using the main runner. If we want to have some sort of way that labels are created for you I'd like that as a separate action, something like `Dug.configure`, whatever. Really some bins to run the runner and and configure things for you would be nice.